app/models/public_users/user.rb

```diff
@@ -10,6 +10,30 @@ class User < Sequel::Model(Sequel[:users].qualify(:public))
 
     attr_accessor :email
 
+    dataset_module do
+      def with_active_stop_press_subscription
+        join(:public__user_subscriptions, user_id: :id)
```

Comment:
Could this be defined by an association?

Reply:
I don't think I can use the association in dataset_module, and this method needs to return the user, so I can't define the whole thing in an association. 
Unless I'm missing something this seems like the best way.